EIPS/eip-5851.md

```diff
@@ -0,0 +1,257 @@
+---
+eip: 5851
+title: On-chain verifiable claims.
+description: Interface for contracts that manage verifiable claims and identifiers as Soulbound tokens.
+author: Yu Liu (@yuliu-debond), Junyi Zhong (@Jooeys)
+discussions-to: https://ethereum-magicians.org/t/eip-5815-kyc-certification-issuer-and-verifier-standard/11513
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-10-18
+requires: 721, 1155, 1167, 1967
+---
+## Abstract
+
+This interface provides a means of issuing and verifying on-chain certifications, specifying the claims as on-chain metadata.
+
+### Definitions
+
+- ZK: Zero-knowledge. Refers to a cryptographic system in which a characteristic of an entity can be proven without revealing which entity the proof refers to.
+
+- SBT: Known as soulbound tokens, these are non-fungible and non transferrable tokens that is used for defining the identity of the users.
```

Comment:
In the `description:` header, you capitalize soulbound. Please choose a consistent capitalization for the entire document.